web/notifications/notifications.go

```diff
@@ -0,0 +1,68 @@
+package notifications
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/cozy/cozy-stack/pkg/apps"
+	"github.com/cozy/cozy-stack/pkg/consts"
+	"github.com/cozy/cozy-stack/pkg/couchdb"
+	"github.com/cozy/cozy-stack/pkg/notifications"
+	"github.com/cozy/cozy-stack/web/jsonapi"
+	"github.com/cozy/cozy-stack/web/middlewares"
+	"github.com/cozy/cozy-stack/web/permissions"
+	"github.com/cozy/echo"
+)
+
+type apiNotif struct {
+	n *notifications.Notification
+}
+
+func (n *apiNotif) ID() string                             { return n.n.ID() }
+func (n *apiNotif) Rev() string                            { return n.n.Rev() }
+func (n *apiNotif) DocType() string                        { return consts.Notifications }
+func (n *apiNotif) Clone() couchdb.Doc                     { return n }
+func (n *apiNotif) SetID(_ string)                         {}
+func (n *apiNotif) SetRev(_ string)                        {}
+func (n *apiNotif) Relationships() jsonapi.RelationshipMap { return nil }
+func (n *apiNotif) Included() []jsonapi.Object             { return nil }
+func (n *apiNotif) Links() *jsonapi.LinksList {
+	return &jsonapi.LinksList{Self: "/notifications/" + n.n.ID()}
+}
+func (n *apiNotif) MarshalJSON() ([]byte, error) {
+	return json.Marshal(n.n)
+}
+
+func createHandler(c echo.Context) error {
+	inst := middlewares.GetInstance(c)
+	n := &notifications.Notification{}
+	if _, err := jsonapi.Bind(c.Request(), &n); err != nil {
+		return err
+	}
+	slug, err := permissions.AllowForWebapp(c, permissions.POST, n)
```

Comment:
Konnectors should be able to create notifications too. In fact, while we don't have background services on server for webapps, it will be mostly konnectors that will create them!